src/batcher.ts

```diff
@@ -141,7 +141,7 @@ export class Batcher {
                 err.message = messages.getMessage('ExternalIdRequired', [sobjectType]);
                 job.emit('error', err);
               }
-              if (err.message.startsWith('Polling time out')) {
+              if (err.name === 'PollingTimeout') {
```

Comment:
this is from the bulk v1 API:
https://github.com/jsforce/jsforce/blob/7c8e6a130a149cc7f310e6628f786f94a98c91b9/src/api/bulk.ts#L707
https://github.com/jsforce/jsforce/blob/7c8e6a130a149cc7f310e6628f786f94a98c91b9/src/api/bulk.ts#L484